File: app/src/main/java/org/bspeice/minimalbible/activity/settings/MinimalBibleSettings.java
package org.bspeice.minimalbible.activity.settings;

import android.os.Bundle;
import android.preference.PreferenceActivity;
import android.support.v7.widget.Toolbar;
import android.widget.LinearLayout;

import org.bspeice.minimalbible.R;
import org.bspeice.minimalbible.activity.BaseActivity;

import butterknife.ButterKnife;
import butterknife.InjectView;

public class MinimalBibleSettings extends PreferenceActivity {

    @InjectView(R.id.toolbar)
    Toolbar toolbar;
    @InjectView(R.id.container)
    LinearLayout root;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        addPreferencesFromResource(R.xml.preferences);
        setContentView(R.layout.activity_settings);
        BaseActivity.setupStatusBar(this);

        ButterKnife.inject(this);

        toolbar.setTitle(R.string.action_settings);
        BaseActivity.setupInsets(this, root);
    }
}

File: app/src/main/kotlin/org/bspeice/minimalbible/activity/settings/MinimalBibleSettings.kt
package org.bspeice.minimalbible.activity.settings

import android.preference.PreferenceActivity
import android.os.Bundle
import org.bspeice.minimalbible.R
import android.support.v7.widget.Toolbar
import org.bspeice.minimalbible.activity.BaseActivity
import android.widget.LinearLayout

/**
 * Created by bspeice on 12/1/14.
 */
class MinimalBibleSettings() : PreferenceActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super<PreferenceActivity>.onCreate(savedInstanceState)

        addPreferencesFromResource(R.xml.preferences)
        setContentView(R.layout.activity_settings)
        BaseActivity.setupStatusBar(this)

        val toolbar = findViewById(R.id.toolbar) as Toolbar
        toolbar.setTitle(R.string.action_settings)

        val root = findViewById(R.id.container) as LinearLayout
        BaseActivity.setupInsets(this, root)
    }
}